packages/salesforcedx-vscode-core/test/commands/commands.test.ts

```diff
@@ -231,7 +232,6 @@ describe('Command Utilities', () => {
         const dirList: string[] = dirPathGatherer.globDirs(
           vscode.workspace.rootPath
         );
-        console.log(`Found dirs: ${dirList}`);
```

Comment:
Remove logging that @guw added. I don't think we need these and there were cluttering up the results when running locally.